docs/standard/data/sqlite/connection-strings.md

```diff
@@ -118,10 +118,10 @@ You can use <xref:Microsoft.Data.Sqlite.SqliteConnectionStringBuilder> as a stro
 
 ### Basic
 
-A basic connection string with a shared cache for improved concurrency.
+A basic connection string.
 
 ```connectionstring
-Data Source=Application.db;Cache=Shared
+Data Source=Application.db
```

Comment:
I think we should leave this one, but add a note:

```
> [!CAUTION]
> Mixing shared-cache mode and write-ahead logging is discouraged. For optimal performance, remove
> `Cache=Shared` when the database is configured to use write-ahead logging.
```

Reply:
Sounds good! Added.